Unstructured-IO/unstructured
# build: script to update all ingest fixtures, add azure ingest fixtures

* Updates CI to install tesseract version 5.3.0 (better than 4.x in various ways incl. perf.).
* Adds azure expected output fixtures for more useful reference points and as a repro for https://github.com/Unstructured-IO/unstructured/issues/346 .
* Adds a script to regenerate ingest test fixtures that is run in an ubuntu docker container (like CI), with the same version of tesseract. See the comments in `scripts/ingest-test-fixtures-update.sh` for details.
* Updates expected outputs with above script.
* Updates individual test-ingest scripts to update expected .json output if  OVERWRITE_FIXTURES=true.

Test instructions:

1. "dirty" some expected output .json's:
```
echo extraline >> test_unstructured_ingest/expected-structured-output/s3-small-batch/small-pdf-set/2023-Jan-economic-outlook.pdf.json
echo extraline >> test_unstructured_ingest/expected-structured-output/biomed-ingest-output-api/65/11/main.PMC6312790.pdf.json
echo extraline >> test_unstructured_ingest/expected-structured-output/biomed-ingest-output-path/07/07/sbaa031.073.PMC7234218.pdf.json
echo extraline >> test_unstructured_ingest/expected-structured-output/azure-blob-storage/IRS-form-1987.pdf.json
echo extraline >> test_unstructured_ingest/expected-structured-output/github-downloadify/test.html.json
```
2. witness that `git status` reflects the changes
3. run `./scripts/ingest-test-fixtures-update.sh` and see that `git status` is now clean with no changes in any files.
